gedditlive/GedditSDK
# Release GedditCommerce version 1.7.1.

# Changelog

- Added: Confetti animation when user answered correctly.
- Changed: Display 3.5 elements when user open promo code list.
- Fixed: Display question popups behind product list and promo code list.

# Testing

In order to test using this branch, you should edit the cocoapods repository used as a source for your Podfile:

1. Make sure that `~/.cocoapods/repos/gedditlive` exists
    - If not, Run `pod install` with the following Podfile:

```ruby
source 'https://github.com/gedditlive/GedditSDK.git'
target 'YOUR_TARGET' do
  use_frameworks!

  pod 'GedditCoreSDK', '1.2.0'
  pod 'GedditCommerceSDK', '1.7.1'
end
```

2. Switch to the `release-1.7.1` branch:

```sh
cd ~/.cocoapods/repos/gedditlive;
git checkout release-1.7.1
```

3. Edit the podspec files to make `pod install` downloads the frameworks on `release-1.7.1` branch.

_~/.cocoapods/repos/gedditlive/GedditCommerceSDK/1.7.1/GedditCommerceSDK.podspec:_
```ruby
s.source = { :http => 'https://github.com/gedditlive/GedditSDK/blob/release-1.7.1/GedditCommerceSDK/1.7.1/GedditCommerceSDK.zip?raw=true'}
```

4. Make sure to clear your pod cache:

```sh
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCommerceSDK
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCoreSDK
```

5. Run `pod install` in your project
